Packages/io.chainsafe.web3-unity/Editor/ABIConverter.cs

```diff
@@ -11,14 +11,14 @@ public class ABIWindow : EditorWindow
     Vector2 TextArea;
 
     // Where is the window menu located
-    [MenuItem("ChainSafe SDK/ABI Converter")]
+    [MenuItem("ChainSafe SDK/ABI Formatter")]
```

Comment:
It's not very obvious by the name what this menu does imo. Maybe something like `Contract class generation`, or `Contract ABI to C#`, or `Convert ABI to C# class` (best one i think), or `Generate ABI wrapper` would work better.